cypress/support/commands.ts

```diff
@@ -24,8 +24,12 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 // Ignoring uncaught exceptions since errors from external apps should not stop de workshop
+import 'cypress-file-upload';
 Cypress.on("uncaught:exception", (err, runnable) => {
     // returning false here prevents Cypress from
     // failing the test
     return false;
 });
+
+
+
```

Comment:
remove the extra blank lines at the end of the file, it is just needed one blank line

Reply:
all done @holgiosalos